cmd/legacy_param_mapper_test.go

```diff
@@ -504,3 +504,53 @@ func TestInvalidClientProtocol(t *testing.T) {
 
 	assert.NotNil(t, err)
 }
+
+func TestEnableEmptyManagedFoldersVetting(t *testing.T) {
```

Comment:
I think we should add this test to cfg package. We can add a happy path test to the existing test to validate Rationalize is called.

Saying this because eventually we are going to get rid of PopulateNewConfigFromLegacyFlagsAndConfig and there will be more migration work at that time.

Reply:
Done. I've added unit tests for Rationalize().